Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -356,6 +358,22 @@ defineSuite([
         });
     });
 
+    it('loads tileset with extras', function() {
+        return Cesium3DTilesTester.loadTileset(scene, tilesetUrl).then(function(tileset) {
+            expect(tileset.extras).toBeDefined();
+            expect(tileset.root.extras).not.toBeDefined();
+
+            var taggedChildren = 0;
+            for(var i = 0;i < tileset.root.children.length; i++) {
```

Comment:
Style things:
* change `for(var` to `for (var` - add space after `for`
* change `i = 0;i` to `i = 0; i` - add space after semicolon
* change `i++` to `++i` - just a convention in the Cesium code base
* save `length` as its own variable before the loop - this convention was originally added for performance reasons since length doesn't have to be re-evaluated every iteration. While it doesn't really for tests it's just good to get into that habit.

```
var length = tileset.root.children.length
for (var i = 0; i < length; ++i) {
```